src/main/plugin/iso19139.ca.HNAP/layout/config-editor.xml

```diff
@@ -509,7 +509,7 @@
                 <gmd:distributionFormat>
                   <gmd:MD_Format>
                     <gmd:name>
-                      <gco:CharacterString>DOC</gco:CharacterString>
+                      <gco:CharacterString></gco:CharacterString>
```

Comment:
Should be able to use `<gco:CharacterString/>` instead of `<gco:CharacterString></gco:CharacterString>`